Mathlib/SetTheory/Nimber/Field.lean

```diff
@@ -218,4 +218,123 @@ instance : CommRing Nimber where
 instance : IsDomain Nimber where
 instance : CancelMonoidWithZero Nimber where
 
+/-! ### Nimber division -/
+
+/-- The nimber inverse `a⁻¹` is recursively defined as the smallest nimber not in the set `s`, which
+itself is recursively defined as the smallest set with `0 ∈ s` and `(1 + (a + a') * b) / a' ∈ s`
+for `0 < a' < a` and `b ∈ s`.
+
+This preliminary definition "accidentally" satisfies `inv 0 = 1`, which the real inverse corrects.
+-/
+def inv (a : Nimber) : Nimber :=
```

Comment:
I'm confused; is this "the nimber inverse" or "an preliminary construction used to define the nimber inverse"? If the latter, can you rename this to `invAux` or `invNonzero` or similar?

Reply:
Well, it's the nimber inverse modulo a different convention for division by zero. I guess by Lean standards this means it's just a preliminary construction, so I'll rename to `invAux`.